src/tests/calculate.test.js

```diff
@@ -0,0 +1,43 @@
+import calculate from '../components/Calculator/calculate.js';
+
+it('Clear the calculator with the AC button', () => {
+  const newValue = 'AC';
+
+  expect(calculate(null, newValue)).toStrictEqual({
+    total: null,
+    next: null,
+    operation: null,
+  });
+});
+
+it('Checking result when pressing "=" button', () => {
+  const mockState = {
+    total: '100',
+    next: '50',
+    operation: '+',
+  };
+
+  const newValue = '=';
+
+  expect(calculate(mockState, newValue)).toStrictEqual({
+    total: '150',
+    next: null,
+    operation: null,
+  });
+});
+
+it('Checking next value when pressing "." button', () => {
+  const mockState = {
+    total: '1',
+    next: '5',
+    operation: null,
+  };
+
+  const newValue = '.';
+
+  expect(calculate(mockState, newValue)).toStrictEqual({
+    total: '1',
+    next: '5.',
+    operation: null,
+  });
+});
```

Comment:
- [ ] Please start with describing to describe what is happening at next.

- [ ] Implement the following tests.

- should throw 0 error if trying to make a zero division
- should throw Zero division error

Reply:
Thanks for the comment!

I hope I did not get this wrong but the two bullet points are about the same issue. Please be more clear.